### PR TITLE
Containerized Docker Application Lifecycle - 01.1 - Chapter 01 - Article 1

### PR DESCRIPTION
**Pull request by mvelosop**
### e-book <--> docs.microsoft.com synchronization
- Chapter 01 - article 1

Related PR: https://github.com/dotnet/docs/pull/9305

### CODE REVIEW

File: docs/standard/containerized-lifecycle-architecture/index.md [hunk 1]
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
 ms.author: wiwagn
 ms.date: 09/22/2017
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/index.md]: change ms.date?

File: docs/standard/containerized-lifecycle-architecture/index.md [hunk 2]
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
...
 ---
-# Introduction to containers and Docker
+# Introduction to Containers and Docker
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/index.md]: Previous title was the correct capitalization

```suggestion
# Introduction to containers and Docker
```

File: docs/standard/containerized-lifecycle-architecture/index.md [hunk 3]
```diff
@@ -5,25 +5,23 @@ author: CESARDELATORRE
 ms.author: wiwagn
```
Comment (mairaw) [inline docs/standard/containerized-lifecycle-architecture/index.md]: can we fix the description here for SEO?

**Review by mairaw — CHANGES_REQUESTED** [review]
Small nits to fix but looks good overall

**Comment by mvelosop** [comment]
@mairaw, we had already fixed metadata in PR https://github.com/dotnet/docs/pull/9305, so I just cherry-picked this commit and applied to a fresh branch (and fixed the title casing)

**Pull request closed without merging**